docs/source/bag-api.rst

```diff
@@ -0,0 +1,32 @@
+API
+===
+
+.. currentmodule:: dask.bag.core
+
+Create Bags
+-----------
+
+.. autosummary::
+   from_sequence
+   from_filenames
+   concat
+
+Turn Bags into other things
+---------------------------
+
+.. autosummary::
+   Bag.to_textfiles
+   Bag.to_dataframe
+
+Bag methods
+-----------
+
+.. autoclass:: Bag
+   :members:
```

Comment:
We might want to do a full autosummary of Bag up at the top.  I suspect that this will make methods more discoverable as well as give a quick at-a-glance summary of available functionality.

```
.. autosummary::
   Bag
   Bag.map
   Bag.filter
   ...
```